src/controller/userController.js

```diff
@@ -47,6 +48,77 @@ class UserController {
       });
     }
   }
+
+  /**
+  * List Authors
+  * Route: GET: /authors
+  * Route: GET: /authors/:search
+  * @param {object} req -Request object
+  * @param {object} res - Response object
+  * @return {res} res - Response object
+  * @memberof USerController
+ */
+  static async getAuthors(req, res) {
+    try {
+      let authors;
+      if (req.params.search) {
+        authors = await Users.findAll({
+          attributes: [
+            'id',
+            'firstName',
+            'lastName',
+            'email',
+            'image',
+            'bio',
+            'username'
+          ],
+          where: {
+            [Op.or]: [
+              {
+                username: {
+                  [Op.like]: `%${req.params.search}%`
+                }
+              },
+              {
+                firstName: {
+                  [Op.like]: `%${req.params.search}%`
+                }
+              },
+              {
+                lastName: {
+                  [Op.like]: `%${req.params.search}%`
+                }
+              },
+            ],
+          }
+        });
+      } else {
+        authors = await Users.findAll({
+          attributes: [
+            'id',
+            'firstName',
+            'lastName',
+            'email',
+            'image',
+            'bio',
+            'username'
+          ],
+          where: {
+            isVerified: true,
+          }
+        });
+      }
+      res.status(200).json({
+        message: 'successfully',
+        authors,
+      });
```

Comment:
Your response message, 'successfully', on line 112 is not self-explanatory. I think something like 'Request completed successfully' is more self-explanatory.

Reply:
```
res.status(200).json({
        success: true,
        ...
});
```